Examples/Algorithms/Digitization/src/DigitizationOptions.cpp

```diff
@@ -47,11 +48,48 @@ void ActsExamples::Options::addDigitizationOptions(
   opt("digi-smear-parameters",
       value<read_range>()->multitoken()->default_value({}),
       "Smearing Input: smear parameters depending on the smearing type, 1 "
-      "parameter for simple gauss, 4 for all others (1 parameter, 2 absolute "
+      "parameter for simple gauss, 3 for all others (1 parameter, 2 absolute "
       "range "
       "values, 1 range indicator: both neg < 0, pos/neg = 0, both pos > 0).");
```

Comment:
Don't we need the fourth one anymore? I can not have a negative value in the read_series, which is why I needed the range indicator

Reply:
Anyhow, we really need that `.json` configured.

Reply:
I think, this was a documentation error to begin with. All smearers that are available take three parameters, not four (simple Gaussian excluded).

Reply:
No, indeed the parameters need 3, but for the command line I need 4.

Reply:
Ah, no I understand. Will fix that.

Reply:
Unfortunately, this shows the limits of the command line implementation.

Reply:
@asalzburger Should be fixed now.